_pages/production.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Production deployment
+lead: >
+  Once you’ve tested your integration in our sandbox environment, you can request deployment to the Login.gov production environment.
+redirect_from:
+  - /production-deployment/
+sidenav:
+  - text: Production endpoints
+    href: "#production-endpoints"
+  - text: Deployments
+    href: "#deployments"
+  - text: Confirm IAA
+    href: "#confirm-iaa"
+  - text: Staging environment
+    href: "#staging-environment"
+  - text: Production configuration
+    href: "#production-configuration"
+  - text: Request deployment
+    href: "#request-deployment"
+  - text: Deploying changes
+    href: "#deploying-changes"
+---
+
+
+## Production endpoints
+
+Our integration documentation includes endpoint urls for our sandbox environment https://idp.int.identitysandbox.gov/. Our production environment is located at https://secure.login.gov/. The URL path to each endpoint remains the same. Only the domain will change. For example, the authorization endpoint will change as follows:
+* OpenID Connect - [https://secure.login.gov/openid_connect/authorize](https://secure.login.gov/openid_connect/authorize)
+* SAML - [https://secure.login.gov/api/saml/auth2020](https://secure.login.gov/api/saml/auth2020)
+
+Please be aware that the IDP certificate (X509Certificate) in the Production environment is different from the IDP certificate in the sandbox environment. The Production IDP certificates can be found here:
+* OpenID Connect - [https://secure.login.gov/api/openid_connect/certs](https://secure.login.gov/api/openid_connect/certs)
+* SAML - [https://secure.login.gov/api/saml/metadata2020](https://secure.login.gov/api/saml/metadata2020)
+
+## Deployments
+
+All changes to integrations between login.gov and your application must be reviewed and deployed. We ask for 1 week notice for new integrations and changes to existing integrations. Regular deployments occur every Thursday by the close of business day. If the regular deployment occurs on a holiday, then it will be completed the following Monday.
+
+## Confirm IAA
+
+You must have a signed IAA (Inter Agency Agreement) in order to deploy to production. You will need to provide the IAA Number this application will be billed under. The IAA number format will include GTC-Order-ModNumber (Example: LGABCFY210001-0001-0000).
```

Comment:
the IAA number formats got broken over a line break, if we code-quote them?, maybe it will stick together?

```suggestion
You must have a signed IAA (Inter Agency Agreement) in order to deploy to production. You will need to provide the IAA Number this application will be billed under. The IAA number format will include `GTC-Order-ModNumber` (Example: `LGABCFY210001-0001-0000`).
```

Reply:
Great suggestions. Thanks @zachmargolis!